src/api/load.test.js

```diff
@@ -102,11 +103,11 @@ describe("api/load", () => {
   it("calls github githubWeightedGraph with the right options", async () => {
     const {options, taskReporter, sourcecredDirectory} = example();
     await load(options, taskReporter);
-    const cacheDirectory = path.join(sourcecredDirectory, "cache");
+    const cache = new DataDirectory(sourcecredDirectory);
```

Comment:
I'm a little surprised this test still works -- in the past the cacheDirectory was a string (value equality) but now we are passing in an object, and the expected DataDirectory will be a different instance than the actual one.

docs are unclear... but I guess this test passing means that `toHaveBeenCalledWith` uses deep equality checking?

Reply:
Yes. It needs to have used deep equality to have worked before this change as well.
`expectedLoadGraphOptions` would not be object equal to the actual object passed.

But indeed the docs are not really clear about it. You can also use any of the matchers with it.
For example, from [`expect.any(constructor)`](https://jestjs.io/docs/en/expect#expectanyconstructor):
> You can use it inside toEqual or toBeCalledWith instead of a literal value.
```js
expect(mock).toBeCalledWith(expect.any(Number));
```